src/redprl/redprl.grm

```diff
@@ -338,6 +338,7 @@ atomicRawTac
 
   | operator LPAREN bindings RPAREN (Ast.$$ (operator, bindings))
   | metavar metavarArgs (Ast.$$# (metavar, metavarArgs))
+  | IDENT (`` IDENT)
```

Comment:
thanks! the reason for having this, by the way, was that there are place where we bind a variable that ranges over a tactic; this happens, for instance, in the fixed point tactical. I don't think this tactical is used in the examples, because in practice you use a fancier tactical which is derived from it, called `REPEAT`.